exercises/practice/bottle-song/test/bottle_song_test.exs

```diff
@@ -0,0 +1,137 @@
+defmodule BottleSongTest do
+  use ExUnit.Case
+
+  describe "[Single stanza]" do
+    # @tag :pending
+    test "First generic stanza" do
+      assert BottleSong.recite(10, 1) == [
+               "Ten green bottles hanging on the wall,",
+               "Ten green bottles hanging on the wall,",
+               "And if one green bottle should accidentally fall,",
+               "There'll be nine green bottles hanging on the wall."
+             ]
```

Comment:
Let's make all of the `recite` return value a single triple-quoted string, multiple stanzas can be separated by an empty line.